config.py

```diff
@@ -270,6 +270,7 @@ def address_from_private_key(private_key):
             private_key = bytes.fromhex(private_key.replace('0x', ''))
         return keys.PrivateKey(private_key).public_key.to_checksum_address()
 
+    ETH_PENDING_TRANSACTION_EXPIRY_SECONDS = config_parser['ETHEREUM'].get('transaction_expiry_seconds') or 30
```

Comment:
pretty sure this needs to be converted from string to int for the ini file

Reply:
My bad! Fixed!

Reply:
them ini files will getcha lol!